home.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CineOnline</title>
+    <link href="css/estilos.css" rel="stylesheet"/>
+</head>
+<body>
+
+<!--Cabecero de la página principal "HOME"-->
+<header>
+    <nav class="nav-fixed">
+        <img id="logo" src="imagenes/cine_adobespark (1).png" alt="Logo Cinemath"/>
+        <h3 id="id">CineOnline</h3>
+        <a href="home.html">Home</a>
+        <a href="movies.html">Películas</a>
+        <a href="home.html">Series</a>
+        <a href="home.html">Precios</a>
+        <a href="home.html">Contacto</a>
+        <a href="home.html" id="slg">Login</a>   

```

Comment:
Los enlaces que no tienen una página aún podemos dejarlos sin ningún destino.
```suggestion
        <a>Series</a>
        <a>Precios</a>
        <a>Contacto</a>
        <a id="slg">Login</a>   
```